style/ruby/README.md

```diff
@@ -1,93 +1,4 @@
 Ruby
 ====
 
-[Sample](sample.rb)
-
-* Avoid conditional modifiers (lines that end with conditionals). [36491dbb9]
-* Avoid multiple assignments per line (`one, two = 1, 2`). [#109]
-* Avoid organizational comments (`# Validations`). [#63]
-* Avoid ternary operators (`boolean ? true : false`). Use multi-line `if`
-  instead to emphasize code branches. [36491dbb9]
-* Avoid explicit return statements. [#48]
```

Comment:
Are we losing this in our move to standard?

Reply:
standard covers this via the Style/RedundantReturn rule